pkgs/applications/science/electronics/kicad/default.nix

```diff
@@ -1,11 +1,15 @@
-{ wxGTK, lib, stdenv, fetchurl, fetchFromGitHub, cmake, libGLU, libGL, zlib
+{ lib, stdenv, fetchFromGitHub, cmake, libGLU, libGL, zlib, wxGTK
 , libX11, gettext, glew, glm, cairo, curl, openssl, boost, pkgconfig
-, doxygen, pcre, libpthreadstubs, libXdmcp
-, wrapGAppsHook
+, doxygen, pcre, libpthreadstubs, libXdmcp, makeWrapper, gnome3
+, gsettings-desktop-schemas, librsvg, hicolor-icon-theme, lndir, cups
+, fetchpatch
+
 , oceSupport ? true, opencascade
+, withOCCT ? false, opencascade-occt
```

Comment:
OCCT should probably be the default (instead of OCE) unless there are good reasons for it, since OCCT is the newer version.

Reply:
OCE does seem outdated...
i was going with OCE because this package used it before i touched it, and KiCad uses it by default
the only reason i added OCCT is because OCE fails to build on aarch64
going by [this](https://forum.freecadweb.org/viewtopic.php?t=12920#p103483), i'm probably going to switch it all over to OCCT, unless i hear from their IRC